evap/staff/templates/staff_evaluation_textanswers_section.html

```diff
@@ -11,18 +11,24 @@
                 <table class="table table-striped{% if not forloop.last %} mb-4{% endif %}">
                     <thead>
                         <tr>
-                            <th style="width: 85%">{% trans 'Text answer' %}</th>
-                            <th style="width: 15%">{% trans 'Publish' %}</th>
+                            <th style="width: 75%">{% trans 'Text answer' %}</th>
+                            <th style="width: 15%"></th>
+                            <th style="width: 10%">{% trans 'Publish' %}</th>
```

Comment:
```suggestion
                            <th style="width: 80%">{% trans 'Text answer' %}</th>
                            <th style="width: 5%"></th>
                            <th style="width: 15%">{% trans 'Publish' %}</th>
```

Reply:
the one-button column can be smaller and the "Publish" column needs a bit more space